Added API/OT status aside for Trust Tokens article (#3919)
GoogleChrome/developer.chrome.com: site/en/docs/privacy-sandbox/trust-tokens/index.md

description: >
  An API to convey a limited amount of information from one browsing context to another (for example, across sites) to help combat fraud, without passive tracking.
date: 2021-05-18
updated: 2022-10-06
authors:
  - samdutton
---

{% Aside %}
The [Trust Tokens origin trial](/origintrials/#/view_trial/2479231594867458049) is now closed.

API status updates will be provided here and on [Chrome Platform Status](https://www.chromestatus.com/feature/5078049450098688).
{% endAside %}

## Implementation status

* [In origin trial](https://web.dev/origin-trials/) Chrome 84 to 101: [now closed](/origintrials/#/view_trial/2479231594867458049).

> * [Demo](https://trust-token-demo.glitch.me/): this is not working, since the origin trial is now closed, but you can still view the code.

Reviewer: ```suggestion
* [Demo](https://trust-token-demo.glitch.me/): this no longer works, as the origin trial is closed. You can still view the code.
```

description: >
  An API to convey a limited amount of information from one browsing context to another (for example, across sites) to help combat fraud, without passive tracking.
date: 2021-05-18
updated: 2022-10-06
authors:
  - samdutton
---

> This article previously referred to Private State Tokens as "Trust Tokens". The Trust Token API has been renamed to the
> Private State Token API to better showcase its privacy and utility benefits.
> 
> Check out [Trust Tokens renamed Private State Tokens](/blog/rename-trust-tokens) for more information.
> 
> API status updates will be provided here and on [Chrome Platform Status](https://www.chromestatus.com/feature/5078049450098688).
> 
> {% endAside %}

Reviewer: ```suggestion
{% Aside %}
This article previously referred to Private State Tokens as "Trust Tokens". The Trust Token
API has been renamed to the Private State Token API to better showcase its privacy and
utility benefits.

Check out [Trust Tokens renamed Private State Tokens](/blog/rename-trust-tokens) for more information.
{% endAside %}
```

## Engage and share feedback

* **Origin trial**: Now [closed](/origintrials/#/view_trial/2479231594867458049).

> * **Demo**: The Trust Tokens origin trial has closed, so the [demo](https://trust-token-demo.glitch.me/) won't work, but you can still view code for trust token issuance and redemption.

Reviewer: ```suggestion
* **Demo**: The Trust Tokens origin trial has closed, so the [demo](https://trust-token-demo.glitch.me/) won't work, but you can still view the code.
```
